server/src/testFixtures/java/io/crate/testing/SymbolAssert.java

```diff
@@ -122,6 +122,14 @@ public SymbolAssert isReference(String expectedName) {
         return this;
     }
 
+    public SymbolAssert isReference(ColumnIdent expectedColumn,
+                                    RelationName expectedRelName,
+                                    DataType<?> expectedType,
+                                    long oid) {
+        assertThat(((Reference) actual).oid()).isEqualTo(oid);
+        return isReference(expectedColumn, expectedRelName, expectedType);
+    }
+
```

Comment:
I think instead of adding a ton of different overloads we should make it more composable.

E.g. we could have something like:

```assertThat(symbol)
        .isReference() // returns `ReferenceAssert`
        .hasOid(oid)
        .hasType(...)
        .hasName(...)
```

Reply:
I will create a separate PR for that (and merge into drop-column branch before this, so that we have hasOid method.